### DISCUSSION TITLE
NodeGraph add support for mysql

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana#49757] [Data Sources]
Greetings,

Would be cool to enable node graph wiht mysql data sources. Data is data and the following data sources satisfy the requirements of the node graph DATA Api.

![image](https://user-images.githubusercontent.com/900951/170744739-02ee990d-94e5-4091-8c31-1e25f92eb0b6.png)

**Comment by yosiasz** [comment]
this does work in 8.5.2 grafana. thanks

**Comment by gabor** [comment]
closing, as functionality is described to work correctly.
